src/routes/index.js

```diff
@@ -1,9 +1,11 @@
 import express, { Router } from "express";
+import twoFactorAuthentication from "./api/2FA.route";
 import home from "./api/home.route";
 import user from "./api/user.route";
 
 const routes = express.Router();
 
 routes.use("/home", home);
 routes.use("/", user);
+routes.use("/", twoFactorAuthentication);
```

Comment:
Add this to user route

Reply:
> Add this to user route

isn't the two factor authenticator an independent route tho ?